### PR TITLE
Harmonize colors on the dashboard

### PR DESCRIPTION
**Pull request by martinvibes**
@djeck1432 Kindly review the changes made. #186

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -16,6 +16,17 @@
 import src.settings
 import src.swap_amm
 
+# color map
+color_map = {
```
Comment (djeck1432) [inline app.py:20]: 1. Constant should CAPITALIZE 
`COLORS_MAP`

**Review by djeck1432 — CHANGES_REQUESTED** [review]

**Comment by martinvibes** [comment]
@djeck1432 kindly review

File: app.py [hunk 2]
```diff
@@ -418,8 +429,8 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
             collateral_usd_amounts,
             values='amount_usd',
             names='token',
-            title='Collateral (USD)',
-            color_discrete_sequence=plotly.express.colors.sequential.Oranges_r,
+            title='Collateral (USD)',  
+            color_discrete_sequence=[color_map.get('Collateral')],
```
Comment (djeck1432) [inline app.py:433]: You changed `COLORS_MAP` but didn't change `color_map`
```suggestion
            color_discrete_sequence=[COLORS_MAP.get('Collateral')],
```

File: app.py [hunk 3]
```diff
@@ -428,7 +439,7 @@ def _get_risk_level(debt_to_supply_ratio: float) -> str:
             values='amount_usd',
             names='token',
             title='Debt (USD)',
-            color_discrete_sequence=plotly.express.colors.sequential.Greens_r,
+            color_discrete_sequence=[color_map.get('Debt')],
```
Comment (djeck1432) [inline app.py:442]: The same here 
```suggestion
            color_discrete_sequence=[COLORS_MAP.get('Debt')],
```

**Review by djeck1432 — CHANGES_REQUESTED** [review]
Please, take a look to this picture, you didn't change color here, still the same:
1. supply and demand in the main chart by their color,
Screenshot 2024-10-03 at 20 31 32
 It should be different color per `supply` `collateral` `debt`
 
 Please, read issue again, there are 3 points which you should complete

**Pull request closed without merging**